spec.html

```diff
@@ -10015,6 +10015,16 @@ <h1>Static Semantics: AssignmentTargetType ( ): ~simple~ or ~invalid~</h1>
         1. Let _expr_ be the |ParenthesizedExpression| that is covered by |CoverParenthesizedExpressionAndArrowParameterList|.
         1. Return the AssignmentTargetType of _expr_.
       </emu-alg>
+      <emu-grammar>
+        CallExpression :
+          CoverCallExpressionAndAsyncArrowHead
+          CallExpression Arguments
+      </emu-grammar>
+      <emu-alg>
+        1. [id="step-assignmenttargettype-web-compat-insertion-point", normative-optional] If the host is a web browser or otherwise supports <emu-xref href="#sec-late-errors-for-function-call-assignment-targets" title></emu-xref> and IsStrict(this |CallExpression|) is *false*, then
```

Comment:
```suggestion
        1. [id="step-assignmenttargettype-web-compat", normative-optional] If the host is a web browser or otherwise supports <emu-xref href="#sec-late-errors-for-function-call-assignment-targets" title></emu-xref> and IsStrict(this |CallExpression|) is *false*, then
```
IDs don't really matter, but this is not an insertion point in the sense used elsewhere because we are not going to replace this step. It's just a step we want to refer to.